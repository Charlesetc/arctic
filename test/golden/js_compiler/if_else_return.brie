define main [
        if true [
            3
        ] else [ 
            2
        ]
        false
]
