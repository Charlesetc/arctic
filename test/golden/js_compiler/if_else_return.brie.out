const global_functions = {1: {}};
function main() {
	if (true) {3;} else {2;};
	return false;
}
console.log(main());